chains/evm/calls/contracts/bridge/bridge_test.go

```diff
@@ -35,10 +47,19 @@ func (s *ProposalStatusTestSuite) SetupTest() {
 	s.mockContractCaller = mock_calls.NewMockContractCallerDispatcher(gomockController)
 	s.mockTransactor = mock_transactor.NewMockTransactor(gomockController)
 	s.bridgeAddress = common.HexToAddress("0x3162226db165D8eA0f51720CA2bbf44Db2105ADF")
+	s.bridgeContract = bridge.NewBridgeContract(s.mockContractCaller, common.HexToAddress(testContractAddress), s.mockTransactor)
+	s.proposal = *proposal.NewProposal(
+		uint8(1),
+		uint64(1),
+		testResourceId,
+		[]byte{},
+		common.HexToAddress(testHandlerAddress),
+		s.bridgeAddress,
+	)
 }
 func (s *ProposalStatusTestSuite) TearDownTest() {}
 
-func (s *ProposalStatusTestSuite) TestProposalStatusFailedContractCall() {
+func (s *ProposalStatusTestSuite) TestProposalStatusFailed_ContractCall() {
```

Comment:
Could you just rename these old tests to reflect the same name structure as new tests as I really like how you structured function (test) names.